src/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.xproj

```diff
@@ -10,7 +10,7 @@
     <ProjectGuid>61b7c351-c77d-43f7-b56f-bb1440178e10</ProjectGuid>
     <RootNamespace>Microsoft.DotNet.Cli.Utils</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin</OutputPath>
```

Comment:
Now that this will actually work should we be forking our output directories between our build and VS projects?

I think there would be some benefits to sticking rid/stage2/bin here. getting the rid would be interesting, but we could use an inline task that just calls PlatformAbstractions.